### PR TITLE
Added Sidebar and button components

### PR DESCRIPTION
**Pull request by swymbnsl**

### CODE REVIEW

**Comment by 0xSaksham** [comment]
@swymbnsl please try removing import react statements from every component, you don't need it

File: src/components/buttons/primary_button.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from "react"
```
Comment (0xSaksham) [inline src/components/buttons/primary_button.jsx]: remove react import

File: src/components/buttons/primary_button.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          : "hover:cursor-pointer hover:bg-[#5e6bfe]") +
+        ` transition-all duration-100 rounded-2xl flex justify-center text-white items-center font-medium bg-[#4956F4]`
+      }
+    >
+      {buttonText}
+    </div>
+  )
+}
+
+// Copy directly:
```
Comment (0xSaksham) [inline src/components/buttons/primary_button.jsx:24]: why is this code commented out?

File: src/components/buttons/secondary_button.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      onClick={clickFunction}
+      style={{ width: width, height: height }}
+      className={`hover:cursor-pointer hover:bg-[#d2d5f6] transition-all duration-100 rounded-2xl flex justify-center items-center font-medium bg-transparent border-2 border-[#4956F4]`}
+    >
+      {buttonText}
+    </div>
+  )
+}
+
+// Copy directly:
```
Comment (0xSaksham) [inline src/components/buttons/secondary_button.jsx:18]: if it is needed then it's okay but try removing this code

File: src/components/shared/sidebar.jsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
+"use client"
+
+import React, { useContext } from "react"
+import logo from "../../../public/logo.svg"
```
Comment (0xSaksham) [inline src/components/shared/sidebar.jsx]: use @ tags to instead of this long path

**Comment by SahilSuman1011** [comment]
@swymbnsl  Brother Stick to the instructions . we want to have good pull requests.

File: src/components/buttons/primary_button.jsx (hunk 1 above)
Comment (swymbnsl) [reply src/components/buttons/primary_button.jsx]: Ok, will do it

File: src/components/buttons/primary_button.jsx (hunk 2 above)
Comment (swymbnsl) [reply src/components/buttons/primary_button.jsx:24]: Just having this reusable snippet to directly copy and paste wherever needed, will remove at the end

File: src/components/shared/sidebar.jsx (hunk 4 above)
Comment (swymbnsl) [reply src/components/shared/sidebar.jsx]: Okay, I'll fix that too

**Pull request merged**